app/app.js

```diff
@@ -428,7 +428,7 @@ angular.module('portainer', [
       }
     })
     .state('service', {
-      url: '^/service/:id/',
+      url: '^/service/:id/:page_anchor',
```

Comment:
Sorry for my late response.

I don't think there is a need for a direct access to a page anchor, what who be the goal here? Who's going to type the full URL to a service (including the service ID) ?

I also think that `location.hash(hash)` set the hash in the URL, we might need to add `id` on elements in the page though.